web/static/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Hacker's Pub",
```

Comment:
It should be `Hackers' Pub` (see where the apostrophe is placed).

Reply:
I fixed it.